main.js

```diff
@@ -23,12 +23,18 @@ const selectors = {
 
 class UI {
    addBook = function (book) {
-     const list = document.createElement('tr');
+     const list = document.createElement('div');
+     list.classList.add('w-100', 'my-2', 'bg-secondary', 'px-3');
      list.innerHTML = `
-    <td>${book.title}</td>
-    <td>${book.author}</td>
-    <td>${book.isbn}</td>
-    <td><button class="delete">Remove</button></td>
+     <div class="d-flex align-items-center justify-content-between w-100">
+     <div class="d-flex w-75 align-items-center text-center">
+    <h5 class="mx-3 w-25">"${book.title}"</h5>
+    <h5 class="mx-3 w-25"> by </h5>
+    <h5 class="mx-3 w-25">${book.author}</h5>
+    <h5 class = "ml-3 w-25"> ISBN: ${book.isbn}</h5>
+    </div>
+  <div> <button class="delete">Remove</button></div>
```

Comment:
- [ ] OPTIONAL
Entries still show up when removed and then the page is refreshed. Kindly endeavor to improve upon this aspect